common/metrics/defs.go

```diff
@@ -788,6 +794,8 @@ const (
 	StorageLatencyTimer
 	// StorageWriteStoreLatency is the latency to write message to store
 	StorageWriteStoreLatency
+	// StorageWriteMessageLatency is the latency to write message to store and ack, but excludes time writing to socket
+	StorageWriteMessageExcludeSocketLatency
```

Comment:
I think `StorageWriteStoreLatency` (above) is what you want. The actual metrics appear to be commented out in code.

```
$ git grep StorageWriteStoreLatency
common/metrics/defs.go:789:     // StorageWriteStoreLatency is the latency to write message to store
common/metrics/defs.go:790:     StorageWriteStoreLatency
common/metrics/defs.go:1012:            StorageWriteStoreLatency:       {Timer, "storage.write.store-latency"},
services/storehost/inConn.go:327:                       // msg.m3Client.RecordTimer(metrics.InConnScope, metrics.StorageWriteStoreLatency, time.Since(tWrite)) // #perfdisable
services/storehost/inConn.go:551:                       // msg.m3Client.RecordTimer(metrics.InConnScope, metrics.StorageWriteStoreLatency, time.Since(tWrite)) // #perfdisable
services/storehost/replicate.go:473:            // msg.m3Client.RecordTimer(metrics.ReplicateExtentScope, metrics.StorageWriteStoreLatency, time.Since(tWrite)) // #perfdisable
```